lib/vscode/src/vs/server/node/connection.ts

```diff
@@ -170,7 +171,7 @@ export class ExtensionHostConnection extends Connection {
 			// While not technically necessary, makes it easier to tell which process
 			// bootstrap-fork is executing. Can also do pkill -f extensionHost
 			// Other spawns in the VS Code codebase behave similarly.
-			[ '--type=extensionHost' ],
+			[ '--type=extensionHost', `--uriTransformerPath=${join(FileAccess.asFileUri('bootstrap-fork', require).fsPath, '../../../../out/node/uri_transformer.js')}` ],
```

Comment:
Should we use the symlinked path here? Then we can use `FileAccess.asFileUri('vs/server/node/uriTransformer', require).fsPath`.

Reply:
This is actually a bit of an ugly hack; could not figure out how to compile with CommonJS inside here, so I moved the file into our `src/` folder where we (already did) compile with CommonJS, and then just load the version we compiled instead of the version VSCode compiled, heh.

Reply:
Added comments to explain this!

Reply:
Right! But you also symlinked it right? So we could just require the symlinked file.